src/test/java/greetznlTestSuite/GreetzFeaturesTest.java

```diff
@@ -0,0 +1,99 @@
+package greetznlTestSuite;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.*;
+import java.util.List;
+
+import static greetznlTestSuite.utils.GreetzConstants.*;
+import static greetznlTestSuite.utils.GreetzHelperMethods.reformatStringToFloat;
+import static greetznlTestSuite.utils.GreetzHelperMethods.takeFromStringOnlyFloat;
+
+public class GreetzFeaturesTest {
+    private WebDriver driver;
+
+    @BeforeClass
+    public void setDriver() {
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver.exe");
+    }
+
+    @BeforeMethod
+    public void login() throws InterruptedException {
+        driver = new ChromeDriver();
+        WebDriverWait wait = new WebDriverWait(driver, 30);
+
+        driver.get(loginLink);
+        driver.manage().window().maximize();
+
+        WebElement loginFormElem = wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("loginForm")));
+
+        loginFormElem.findElement(By.name("email")).sendKeys(email);
+        loginFormElem.findElement(By.name("password")).sendKeys(pass);
+        driver.findElement(By.id("login-cta")).click();
+        Thread.sleep(2000);
+    }
+
+    @AfterMethod
+    public  void closeDriver(){
+        driver.close();
```

Comment:
Just a small hint. close() closes current tab and quit() quits whole instance.
So you can write quit() and it's more reliable ( of course if you quit whole browser instance ).